modules/project/main.tf

```diff
@@ -0,0 +1,46 @@
+locals {
+  project_folder = coalesce(var.custom_project_folder, var.project_name)
+}
+
+module "workspace" {
+  source = "../workspaces"
+
+  count = var.enabled ? 1 : 0
+
+  auto_apply            = false // TODO: verify
```

Comment:
feature flag?

Reply:
@osterman I went ahead and made this configurable at the project level via the YAML config. Let me know if that's what you had in mind 👍